### PR TITLE
changed dropdown design

### PR DESCRIPTION
**Pull request by fakhruddinkw**
This is regarding: https://github.com/Real-Dev-Squad/todo-action-items/issues/151

**OLD**
old-drop-down-welcome

**NEW**
welcome-new

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -106,15 +106,21 @@ nav li {
...
-  color: var(--color-black);
+  width: 150px;
+  border-radius: 10px;
+  color: #000;
   position: absolute;
-  top: 85px;
-  right: 16px;
+  top: 88px;
+  right: 37px;
   z-index: 100;
```
Comment (Pratiyushkumar) [inline css/navbar.css:115]: kindly put this in rem units and not in px

File: css/navbar.css [hunk 2]
```diff
@@ -123,33 +129,38 @@ nav li {
...
   width: 100%;
-  height: 100%;
+  font-weight: 700;
   background-color: var(--color-white);
   color: var(--color-black);
   border: none;
-  padding: 1rem;
-  font-size: 1rem;
+  line-height: 19.575px;
+  padding: 15px 30px;
```
Comment (Pratiyushkumar) [inline css/navbar.css:154]: here as well

File: index.html [hunk 3]
```diff
@@ -113,7 +133,7 @@
           <div class="nav-login">
             <a
               class="btn-login"
-              href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+              href="https://github.com/login/oauth/authorize?client_id=8b97fd58a86f1d06b0e2"
```
Comment (Pratiyushkumar) [inline index.html:136]: why are we hard coding the client id, is it a good practice...
can we please find a better solution for it?

**Review by Pratiyushkumar — COMMENTED** [review]
where is the  test cases??

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

File: css/navbar.css [hunk 4]
```diff
@@ -280,12 +291,19 @@ nav li a:hover {
   }
 }
 
+
+@media (max-width: 1023px) {
```
Comment (TanishqSingla) [inline css/navbar.css:295]: What is this dimension based on?

File: index.html (hunk 3 above)
Comment (TanishqSingla) [inline index.html:136]: This client id is being used as development environment id in the javascript file, please confirm this url

**Review by TanishqSingla — CHANGES_REQUESTED** [review]
Added some comments, please check

File: index.html (hunk 3 above)
Comment (fakhruddinkw) [reply index.html:136]: good catch. Thank you

File: css/navbar.css (hunk 4 above)
Comment (fakhruddinkw) [reply css/navbar.css:295]: based on where it breaks @TanishqSingla

**Review by sahsisunny — APPROVED** [review]

**Pull request merged**